internal/db_impl/sqlite3/wrappers.go

```diff
@@ -0,0 +1,128 @@
+package sqlite3
+
+import (
+	"context"
+	"database/sql"
+
+	"github.com/sirupsen/logrus"
+)
+
+// Collection of wrappers to help with tracing and debugging of SQL queries and statements.
+
+type QueryWrapper interface {
+	QueryContext(ctx context.Context, query string, args ...any) (*sql.Rows, error)
+	QueryRowContext(ctx context.Context, query string, args ...any) *sql.Row
+	ExecContext(ctx context.Context, query string, args ...any) (sql.Result, error)
+	PrepareStatement(ctx context.Context, query string) (StmtWrapper, error)
+}
+
+type StmtWrapper interface {
```

Comment:
Can you add explanation in documentation what situations are for query and for statement?

Reply:
This is a direct match go go's Queries and Statements https://go.dev/doc/database/prepared-statements

Reply:
I have added some comments as requested.